firefox-ios/Client/Application/ImageIdentifiers.swift

```diff
@@ -20,6 +20,7 @@ public struct ImageIdentifiers {
     public static let homeHeaderLogoBall = "fxHomeHeaderLogoBall"
     public static let homeHeaderLogoText = "fxHomeHeaderLogoText"
     public static let homepagePocket = "homepage-pocket"
+    public static let locationAddressAutofillIcon = "location"
```

Comment:
This is an icon that is in our standard image library Acorn and therefor this image should be added to `StandardImageIdentifiers` instead of `ImageIdentifiers.swift`. If you downloaded the image from Figma, this image should also be re-added to make sure it matches what we have in Acorn and the name has to be adjusted.